### ISSUE TITLE
Building fails

### ISSUE THREAD

**Issue by krt16s** [mbmb5/Eylca#1]
Building fails for me. Most likely the error gets silently ignored when you are running AndroidStudio.

````
:app:lintVitalRelease/home/krt/src/fdroiddata/build/mbmb5.lumixextendedcontrolapp/app/src/main/java/mbmb5/extendedcontrolapp/StreamViewManaging.java:47: Error: Suspicious method call; should probably call "draw" rather than "onDraw" [WrongCall]
                    streamView.onDraw(canvas);
                               ~~~~~~

   Explanation for issues of type "WrongCall":
   Custom views typically need to call measure() on their children, not
   onMeasure. Ditto for onDraw, onLayout, etc.

1 errors, 0 warnings
 FAILED

FAILURE: Build failed with an exception.

````

**Comment by mbmb5** [comment]
Ok thanks
Indeed, android studio continues building the app even with the error.
I will update the code.
Thanks

**Comment by mbmb5** [comment]
Solved by 21b3a87448edc03f4fd35dcdf406b0c96b3f85ba
Other issues of same type solved in 489ea317594da1956e990885547a1873fc8210ac
v0.1-alpha2 should compile without error
